tests/backend_test.py

```diff
@@ -928,4 +928,6 @@ def test_qir_conversion(authenticated_quum_backend: QuantinuumBackend) -> None:
     c = b.get_compiled_circuit(c0)
     h = b.process_circuit(c, n_shots=10, language=Language.QIR)  # type: ignore
     r = b.get_result(h)
-    assert len(r.get_shots()) == 10
+    shots = r.get_shots()
+    assert len(shots) == 10
+    assert all(len(shot) == 2 for shot in shots)
```

Comment:
What would be the expected result for a circuit containing more than one classical register? Do we want to add tests for that?

Reply:
I'd expect it to work, but we cannot test it at the moment becaiuse of https://github.com/CQCL/pytket-qir/blob/f854760ec4d9ed868bfd6de2d8978f1718ba082c/pytket/qir/conversion/api.py#L88 .

Reply:
Good point, we should keep that in mind and add a test after this compatibility issues are resolved.